.gitignore

```diff
@@ -19,6 +19,9 @@ out/
 *.iml
 *.ipr
 
+ReadMe.txt
```

Comment:
Can you explain what these files (ReadMe.txt and context.xml) are, and why we want to ignore them?